### ISSUE TITLE
Issue with Seq-Cifar100

### ISSUE THREAD

**Issue by Damonly15** [aimagelab/mammoth#41]
Hi

I think there might be an issues with the latest implementation of the seq-cifar100 dataset. Up to the second task everything seems normal. But for some reason, the validation accuracy for task 3 and all later task stays around 0 all the time. I tried der with optimal hyper parameters. Training loss looks normal and performance on task 1 & 2 too. Maybe there is something wrong with the test loader?
